### PR TITLE
fix: ignore console instance on config meta

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: lib/loader/mixin/config.js [hunk 1]
```diff
@@ -104,6 +104,11 @@ module.exports = {
 function setConfig(obj, filepath) {
   for (const key of Object.keys(obj)) {
     const val = obj[key];
+    // ignore console
+    if (key === 'console' && val && typeof val.Console === 'function' && val.Console === Console) {
```
Comment (fengmk2) [inline lib/loader/mixin/config.js]: 顺便修复一个 bug。。。

**Comment by Maledong** [comment]
@fengmk2 : It seems good for Nodejs 12 :D

**Comment by Maledong** [comment]
When egg-core is released, maybe Gao's problem is fixed as well.

**Comment by Maledong** [comment]
@atian25: codecov/project is frozen.

**Pull request merged**